2023-11-02 14:13:34,584 INFO: Microblog startup [in /Users/gavino/ifa/tutorial/app/__init__.py:29]
2023-11-02 14:13:42,449 INFO: Microblog startup [in /Users/gavino/ifa/tutorial/app/__init__.py:29]
2023-11-02 14:49:51,510 INFO: Microblog startup [in /Users/gavino/ifa/tutorial/app/__init__.py:29]
2023-11-02 15:19:04,101 INFO: Microblog startup [in /Users/gavino/ifa/tutorial/app/__init__.py:29]
2023-11-02 15:31:10,565 INFO: Microblog startup [in /Users/gavino/ifa/tutorial/app/__init__.py:29]
2023-11-02 15:31:14,297 ERROR: Exception on /index [GET] [in /Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/gavino/ifa/tutorial/app/routes.py", line 22, in index
    return render_template('index.html', title='Home', form=form, posts=posts)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 151, in render_template
    return _render(app, template, context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 132, in _render
    rv = template.render(context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "/Users/gavino/ifa/tutorial/app/templates/base.html", line 36, in top-level template code
    {% block content %}{% endblock %}
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 19, in block 'content'
    {% include '_post.html' %}
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: _post.html
2023-11-02 15:31:16,186 ERROR: Exception on /explore [GET] [in /Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/gavino/ifa/tutorial/app/routes.py", line 131, in explore
    return render_template('index.html', title='Explore', posts=posts)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 151, in render_template
    return _render(app, template, context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 132, in _render
    rv = template.render(context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "/Users/gavino/ifa/tutorial/app/templates/base.html", line 36, in top-level template code
    {% block content %}{% endblock %}
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 19, in block 'content'
    {% include '_post.html' %}
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: _post.html
2023-11-02 15:31:23,208 ERROR: Exception on /index [GET] [in /Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/gavino/ifa/tutorial/app/routes.py", line 22, in index
    return render_template('index.html', title='Home', form=form, posts=posts)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 151, in render_template
    return _render(app, template, context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 132, in _render
    rv = template.render(context)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 2, in top-level template code
    {% extends "base.html" %}
  File "/Users/gavino/ifa/tutorial/app/templates/base.html", line 36, in top-level template code
    {% block content %}{% endblock %}
  File "/Users/gavino/ifa/tutorial/app/templates/index.html", line 19, in block 'content'
    {% include '_post.html' %}
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/gavino/ifa/tutorial/myenv/lib/python3.10/site-packages/flask/templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: _post.html
